src/app/Navbar.tsx

```diff
@@ -159,7 +161,14 @@ async function _connect(props: INavbarProps, web3: Web3) {
   const accounts = await kit.web3.eth.getAccounts()
   kit.defaultAccount = accounts[0]
 
-  managerContract = new kit.web3.eth.Contract(managerABI.abi as unknown as AbiItem, constants.ManagerContractAddress)
+  managerContract = new kit.web3.eth.Contract(
+    managerABI.abi as AbiItem[],
+    constants.addresses.Manager,
+  ) as unknown as Manager
```

Comment:
now we can use the types in `src/types`, so it's easier to call the contracts functions